packages/firebase_auth/firebase_auth/ios/firebase_auth/Sources/firebase_auth/FLTFirebaseAuthPlugin.m

```diff
@@ -602,51 +602,71 @@ static void handleAppleAuthResult(FLTFirebaseAuthPlugin *object, AuthPigeonFireb
     return;
   }
   if (credentials) {
-    [auth signInWithCredential:credentials
-                    completion:^(FIRAuthDataResult *authResult, NSError *error) {
-                      if (error != nil) {
```

Comment:
The whole indentation changed because of this which makes reviewing harder, why formatting changed?